src/broker/broker_config.c

```diff
@@ -170,6 +170,69 @@ static const char *tbl_conf_err_msg[] = {
 #define MAX_NUM_OF_CONF_FILE_LOADED     5
 static char *conf_file_loaded[MAX_NUM_OF_CONF_FILE_LOADED];
 
+/* The order of keywords is based on the manual. */
+const char *broker_keywords[] = {
+  "ACCESS_CONTROL",
+  "ACCESS_CONTROL_FILE",
+  "ADMIN_LOG_FILE",
+  "MASTER_SHM_ID",
+  "ACCESS_LIST",
+  "ACCESS_MODE",
+  "BROKER_PORT",
+  "CONNECT_ORDER",
+  "ENABLE_MONITOR_HANG",
+  "KEEP_CONNECTION",
+  "MAX_NUM_DELAYED_HOSTS_LOOKUP",
+  "PREFERRED_HOSTS",
+  "RECONNECT_TIME",
+  "REPLICA_ONLY",
+  "APPL_SERVER_MAX_SIZE",
+  "APPL_SERVER_MAX_SIZE_HARD_LIMIT",
+  "APPL_SERVER_PORT",
+  "APPL_SERVER_SHM_ID",
+  "AUTO_ADD_APPL_SERVER",
+  "MAX_NUM_APPL_SERVER",
+  "MIN_NUM_APPL_SERVER",
+  "TIME_TO_KILL",
+  "CCI_DEFAULT_AUTOCOMMIT",
+  "LONG_QUERY_TIME",
+  "LONG_TRANSACTION_TIME",
+  "MAX_PREPARED_STMT_COUNT",
+  "MAX_QUERY_TIMEOUT",
+  "SESSION_TIMEOUT",
+  "STATEMENT_POOLING",
+  "JDBC_CACHE",
+  "JDBC_CACHE_HINT_ONLY",
+  "JDBC_CACHE_LIFE_TIME",
+  "TRIGGER_ACTION",
+  "ACCESS_LOG",
+  "ACCESS_LOG_DIR",
+  "ACCESS_LOG_MAX_SIZE",
+  "ERROR_LOG_DIR",
+  "LOG_DIR",
+  "SLOW_LOG",
+  "SLOW_LOG_DIR",
+  "SQL_LOG",
+  "SQL_LOG_MAX_SIZE",
+  "SERVICE",
+  "SSL",
+  "SOURCE_ENV",
+  /* Below is a keyword referenced from the source code, although it is not in the manual. */
+  "APPL_SERVER",
+  "CACHE_USER_INFO",
+  "CCI_PCONNECT",
+  "DATABASES_CONNECTION_FILE",
+  "ENABLE_MONITOR_SERVER",
+  "JDBC_CACHE_ONLY_HINT",
```

Comment:
It will be modified earlier, so you will remove this keyword (JDBC_CACHE_ONLY_HINT).

Reply:
It seems to be missing while moving the code from ini_parser.c to broker_config.c
Thank you for your comment!

Reply:
I'll take JDBC_CACHE_HINT_ONLY as a broker keyword.